src/components/molecules/registerModal.jsx

```diff
@@ -11,15 +11,13 @@ function RegisterModal({ isOpen, handleClose }) {
 
   const handleRegisterSubmit = (e) => {
     e.preventDefault();
-    console.log(username);
-    console.log(password);
     register(username, password);
     handleClose();
   };
 
   return (
     isOpen && (
-      <div className="absolute top-0 left-0 flex justify-center items-center h-full w-full bg-black bg-opacity-25 z-30">
+      <div className="absolute top-0 left-0 flex justify-center items-center h-[100vh] w-full bg-black bg-opacity-25 z-30">
```

Comment:
You don’t necessarily have to change it, but I thought changing absolute to fixed would provide a better user experience by keeping the modal fixed in place relative to scrolling

Reply:
Yes totally agree, that's already in my next branch!